src/commands/add.js

```diff
@@ -67,8 +67,24 @@ Examples:
 
   // find out the type of install
   const specs = detectInstall(context)
-  const { moduleName } = specs
-  const modulePath = `${process.cwd()}/node_modules/${moduleName}`
+  const { moduleName, global } = specs
+  let modulePath
+  if (global) {
```

Comment:
`global` is determined in `detectInstall`. Only `ignite-library` will have this flag set to true.